src/main/java/com/powsybl/openloadflow/sa/ContingencyParametersJsonSerializer.java

```diff
@@ -0,0 +1,60 @@
+package com.powsybl.openloadflow.sa;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.google.auto.service.AutoService;
+import com.powsybl.commons.extensions.ExtensionJsonSerializer;
+import com.powsybl.commons.json.JsonUtil;
+import com.powsybl.contingency.Contingency;
+
+import java.io.IOException;
+
+@AutoService(ExtensionJsonSerializer.class)
+public class ContingencyParametersJsonSerializer implements ExtensionJsonSerializer<Contingency, ContingencyParameters> {
+
+    @Override
+    public String getExtensionName() {
+        return "contingency-parameters";
+    }
+
+    @Override
+    public String getCategoryName() {
+        return "contingency";
+    }
+
+    @Override
+    public Class<? super ContingencyParameters> getExtensionClass() {
+        return ContingencyParameters.class;
+    }
+
+    /**
+     * Specifies serialization for our extension: ignore name et extendable
```

Comment:
Write "and" instead of "et" ?

Reply:
fixed in a8f0dabbdba0b462b430a614e05b568d1ce58a14